[2022-11-27T09:42:36.585+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.is_api_available scheduled__2022-11-26T00:00:00+00:00 [queued]>
[2022-11-27T09:42:36.644+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.is_api_available scheduled__2022-11-26T00:00:00+00:00 [queued]>
[2022-11-27T09:42:36.647+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2022-11-27T09:42:36.649+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2022-11-27T09:42:36.651+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2022-11-27T09:42:36.754+0000] {taskinstance.py:1383} INFO - Executing <Task(HttpSensor): is_api_available> on 2022-11-26 00:00:00+00:00
[2022-11-27T09:42:36.789+0000] {standard_task_runner.py:55} INFO - Started process 4637 to run task
[2022-11-27T09:42:36.859+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'is_api_available', 'scheduled__2022-11-26T00:00:00+00:00', '--job-id', '10', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmpsrbgllba']
[2022-11-27T09:42:36.901+0000] {standard_task_runner.py:83} INFO - Job 10: Subtask is_api_available
[2022-11-27T09:42:37.226+0000] {task_command.py:376} INFO - Running <TaskInstance: user_processing.is_api_available scheduled__2022-11-26T00:00:00+00:00 [running]> on host c7f7231a8603
[2022-11-27T09:42:37.406+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=is_api_available
AIRFLOW_CTX_EXECUTION_DATE=2022-11-26T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-11-26T00:00:00+00:00
[2022-11-27T09:42:37.412+0000] {http.py:120} INFO - Poking: api/
[2022-11-27T09:42:37.462+0000] {base.py:71} INFO - Using connection ID 'user_api' for task execution.
[2022-11-27T09:42:37.494+0000] {http.py:148} INFO - Sending 'GET' to url: https://randomuser.me/api/
[2022-11-27T09:42:38.281+0000] {base.py:213} INFO - Success criteria met. Exiting.
[2022-11-27T09:42:38.442+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=is_api_available, execution_date=20221126T000000, start_date=20221127T094236, end_date=20221127T094238
[2022-11-27T09:42:38.631+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2022-11-27T09:42:38.799+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
